Interfaces/GCHP/gchp_chunk_mod.F90

```diff
@@ -895,8 +895,10 @@ SUBROUTINE GCHP_Chunk_Run( GC,                                             &
                                     RC             = RC         )
 
     ! Pass time values obtained from the ESMF environment to HEMCO
+#if !defined( MODEL_GEOS )
```

Comment:
Why is this bypassed for GEOS?

Reply:
SetHcoTime is also called in hco_interface_mod.F90 [https://github.com/GEOS-ESM/geos-chem/blob/main/GeosCore/hco_interface_gc_mod.F90#L897](https://github.com/GEOS-ESM/geos-chem/blob/main/GeosCore/hco_interface_gc_mod.F90#L897). Calling it twice can screw up some of the first and rewind flags, so I'm skipping it here in GEOS. This could probably be removed for GCHP as well but I haven't tested it.

Reply:
Okay, I'll look into this and test. Making the handling the same for both GEOS and GCHP is always good.